### PR TITLE
Tweak issue template and reorganize FAQ

### LINKED ISSUE #1860: Update GitHub issue template
The template doesn't say to try the latest version.

### PR DESCRIPTION
**Pull request by briandealwis**
Fixes #1860.

Tweaked the ISSUE_TEMPLATE to:
  - add note to verify with latest version, and shortened the header by directing to the FAQ
  - move the environment section to the top and include explicit call outs to Jib version and build tool info

Reorganized the FAQ:
- grouped questions relating to about-images, how-tos and common-problems. 
- made the _How can I diagnose problems pulling or pushing from remote registries?_ question into a central registry-errors section.
- fixed level of the _Where is bash?_ question
- extra directory -> extra directories

### CODE REVIEW

File: docs/faq.md [hunk 1]
```diff
@@ -83,6 +89,63 @@ jib.from.image = 'gcr.io/distroless/java:debug'
...
+
+See [Extended Usage](../jib-maven-plugin#extended-usage) for the `<container><format>` configuration.
+
+#### Gradle
+
+See [Extended Usage](../jib-gradle-plugin#extended-usage) for the `container.format` configuration.
+
+### Why is my image created 48+ years ago?
+
+For reproducibility purposes, Jib sets the creation time of the container images to 0 (January 1st, 1970). If you would like to forgo reproducibility and use the real creation time, set the `useCurrentTimestamp` parameter to `true` in your build configuration.
```
Comment (loosebazooka) [inline docs/faq.md]: 0 -> 0+1?
Or epoch + 1

File: docs/faq.md [hunk 2]
```diff
@@ -83,6 +89,63 @@ jib.from.image = 'gcr.io/distroless/java:debug'
...
+
+#### Maven
```
Comment (loosebazooka) [inline docs/faq.md]: should these point to the subproject configs?

File: .github/ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -1,33 +1,32 @@
 <!--
-Before filing an issue, please check our Frequently Asked Questions:
+Before filing an issue, please verify the issue still occurs with the latest version of Jib.
```
Comment (loosebazooka) [inline .github/ISSUE_TEMPLATE.md]: Should we also leave the FAQ thing in there?

File: .github/ISSUE_TEMPLATE.md [hunk 4]
```diff
@@ -1,33 +1,32 @@
...
 -->
 
-**Description of the issue**:
+**Environment**:
 
-<!--
-If you are encountering errors pulling or pushing to remote registries,
-please check our FAQ before filing an issue.  Specifically:
+- *Jib version:*
+- *Build tool:* <!-- please include the version -->
```
Comment (loosebazooka) [inline .github/ISSUE_TEMPLATE.md]: Isn't version specified above?

Comment (briandealwis) [reply .github/ISSUE_TEMPLATE.md]: This is supposed to be the Maven or Gradle version; I'll make that more explicit.

File: .github/ISSUE_TEMPLATE.md (hunk 3 above)
Comment (briandealwis) [reply .github/ISSUE_TEMPLATE.md]: I didn't analyze the incoming issues, but it seems most that are FAQ'd are relating to registry errors.  Which is covered by the next part.

**Comment by chanseokoh** [comment]
Didn't look into shuttling FAQ, but I just wanted to make sure that old links continue to work.

**Comment by briandealwis** [comment]
Yes — I just shifted the contents.  The following shows any differences in header text.
```
$ grep '^#' docs/faq.md | sort > new
$ git show origin/master:docs/faq.md 2>&1 | grep '^#' | sort > old
$ diff old new
6a7,8
> ## Common Problems
> ## Configuring Jib
8a11
> ### <a name="registry-errors"></a>How can I diagnose problems pulling or pushing from remote registries?
14d16
< ### How can I diagnose problems pulling or pushing from remote registries?
30a33
> ### Where is bash?
49d51
< #### Where is bash?
```

_Common Problems_ and _Configuring Jib_ are new grouping headers.

**Pull request merged**